iphone/Classes/APSHTTPClient/APSHTTPRequest.h

```diff
@@ -20,6 +20,7 @@ typedef NS_ENUM(NSInteger, APSRequestError) {
 @protocol APSConnectionDelegate <NSURLConnectionDelegate, NSURLSessionDelegate, NSURLSessionDataDelegate>
 @optional
 -(BOOL)willHandleChallenge:(NSURLAuthenticationChallenge *)challenge forConnection:(NSURLConnection *)connection;
```

Comment:
This should not be required anymore. Also delegates like `NSURLConnectionDelegate` should be replaced. Please make sure to test all `Ti.Network.HTTPClient` API's in your unit-tests and ensure the old ones work without adjustments.